CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -111,5 +117,74 @@ public void CanRemindLateDevelopersAtTenThirtyOnFriday()
                 _slackApi.ReceivedRequests[i].Url.Should().Be(SlackApiAddress + SlackApiPostMessagePath);
             }
         }
+        
+        
+        [TestFixture]
+        public class BetweenTenThirtyAndOneThirty
+        {
+            [OneTimeSetUp]
+            public void Init()
+            {
+                HandleSetUp();
+            }
+            
+            [Test]
+            [TestCase(10, 45, 0)]
+            [TestCase(11, 00, 4)]
+            [TestCase(11, 15, 4)]
+            [TestCase(11, 30, 8)]
+            [TestCase(11, 45, 8)]
+            [TestCase(12, 00, 12)]
+            [TestCase(12, 15, 12)]
+            [TestCase(12, 30, 16)]
+            [TestCase(12, 45, 16)]
+            [TestCase(13, 00, 20)]
+            [TestCase(13, 15, 20)]
+            public void CanRemindLateDevelopersEveryHalfHourUntilOneThirty(int hour, int minute, int expectedCount)
+            {                      
+                var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
+                var clock = new ClockStub(
+                    new DateTimeOffset(
+                        new DateTime(2019, 03, 01, hour, minute, 0)
+                    )
+                );
+
+                var remindLateDevelopers = new RemindLateDevelopers(getLateDevelopers, _remindDeveloper, clock);
+
+                remindLateDevelopers.Execute(new RemindLateDevelopersRequest
+                    {
+                        Message = "Please make sure your timesheet is submitted by 13:30 on Friday."
+                    }
+                );
+               
+                _slackApi.ReceivedRequests.Count.Should().Be(expectedCount);
+            }  
+        }
+
+        [Test]
+        public void CanRemindLearnTechChannelAtOneThirty()
+        {
+            var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
+            
+            var clock = new ClockStub(
+                new DateTimeOffset(
+                    new DateTime(2019, 03, 01, 13, 30, 0)
+                )
+            );
+
+            var shameLateDevelopers = new ShameLateDevelopers(getLateDevelopers, _remindDeveloper, clock);
+            
+            shameLateDevelopers.Execute(new ShameLateDevelopersRequest
+                {
+                    Message = ":bell: give me the sheets yo :bell:",
```

Comment:
Is this message what we would expect to be sent to the end users?

Reply:
I think we should extract a variable for this as we assert on it as well.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/17/commits/4c1e2a6d7689b008f4b7b1ae729a7f0d46cc5870.